sources/api/netdog/src/net_config.rs

```diff
@@ -0,0 +1,197 @@
+//! The net_config module contains the strucures needed to deserialize a `net.toml` file.
+//!
+//! These structures are the user-facing options for configuring one or more network interfaces.
+use crate::interface_name::{deserialize_interfaces_map, InterfaceName};
+use indexmap::IndexMap;
+use serde::Deserialize;
+use snafu::{ensure, OptionExt, ResultExt};
+use std::{fs, path::Path};
+
+pub(crate) static NET_CONFIG: &str = "/var/lib/bottlerocket/net.toml";
```

Comment:
```suggestion
pub(crate) static DEFAULT_NET_CONFIG_FILE: &str = "/var/lib/bottlerocket/net.toml";
```

I also think it'd make more sense to define this in `main.rs` where it's used, rather than making it part of the module's interface to specify a default.